### PR TITLE
Add screen and components

### PR DESCRIPTION
**Pull request by AangelF**
# Description
-Add screen Customer Service
-Add screen Opinion
-Add component QualifyButton
-Add component Opinion
-Fix details
  
  ## Screenshots 
screen2
screen3

### CODE REVIEW

**Review by LuisGomez89 — APPROVED** [review]

**Review by Namoku — APPROVED** [review]

**Review by LuisGomez89 — APPROVED** [review]

**Review by xManuel007 — APPROVED** [review]

File: src/components/molecules/ServiceGeneralCard/styled.js [hunk 1]
```diff
@@ -15,4 +15,4 @@ export const GroupColumn = styled.View`
 `
 export const ShadowView = styled.View`
   box-shadow: 0px 8px 10px rgba(0, 0, 0, 0.2); //Adri favor de quitarselo si
```
Comment (YilKanda) [inline src/components/molecules/ServiceGeneralCard/styled.js:17]: There should be no feedback to team members on GitHub

File: src/screens/Recruiter/ServicesHistory.js [hunk 2]
```diff
@@ -20,13 +20,19 @@ export const ServicesHistory = () => {
...
-    
+        {/* AQUI DEBE IR CON MAPPING/FLATLIST CON LAS LECTURAS A FIREBASE Y QUE SE GENEREN LOS UPCOMINGSERVICECOMPONENTS NECESARIOS */}
+        <ServiceGeneralCard
+          servicio={'Fontanero'}
+          status={'SERVICIO FINALIZADO'} // esto lo recibe como prop y utiliza la cadena "SERVICIO FINALIZADO" para decidir si muestra o no el boton de estrella para calificar.
+        />
+ {/*        <ServiceGeneralCard
+          servicio={'Fontanero'}
+          status={'SERVICIO ACEPTADO'}
+        /> */}
```
Comment (YilKanda) [inline src/screens/Recruiter/ServicesHistory.js:31]: Code between comments that is not used is best removed.

File: src/components/molecules/QualifyButton/index.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+          quantity={1}
+          startValue={1}
+          readBoolean={true}
+          backgroundColor={'hover'}
+        />
+      </TextCenter>
+      <GeneralText color= 'secondary'weight title={'Calificar'} size={'h7'} />
+    </StarPosition>
+  );
+};
```
Comment (YilKanda) [inline src/components/molecules/QualifyButton/index.js:20]: Add a blank line

File: src/components/molecules/QualifyButton/index.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import { View, Text, Touchable } from 'react-native';
+import React from 'react';
+import { GeneralText, StarRating } from '../../atoms';
+import { StarPosition, TextCenter } from './styled';
+
+export const QualifyButton = () => {
+  return (
+    <StarPosition onPress={()=>console.log("Hola")/* abrir el pop up de del componente para escribir la opinion */}  >
```
Comment (YilKanda) [inline src/components/molecules/QualifyButton/index.js:8]: Don't use console.log() on GitHub, you can use [Alert.alert()](https://reactnative.dev/docs/alert) instead

File: src/screens/Recruiter/ServicesHistory.js [hunk 5]
```diff
@@ -20,13 +20,19 @@ export const ServicesHistory = () => {
...
       />
       <ScrollView>
-        {/* AQUI DEBE IR CON MAPPING CON LAS LECTURAS A FIREBASE Y QUE SE GENEREN LOS UPCOMINGSERVICECOMPONENTYS NECESARIOS */}
-  
-        <ServiceGeneralCard servicio={'Fontanero'}  status={'SERVICIO FINALIZADO'}/>
-        <ServiceGeneralCard servicio={'Fontanero'}  status={'SERVICIO ACEPTADO'}/>
-        <ServiceGeneralCard servicio={'Fontanero'}  status={'SERVICIO RECHAZADO'}/>
-     
-    
+        {/* AQUI DEBE IR CON MAPPING/FLATLIST CON LAS LECTURAS A FIREBASE Y QUE SE GENEREN LOS UPCOMINGSERVICECOMPONENTS NECESARIOS */}
```
Comment (YilKanda) [inline src/screens/Recruiter/ServicesHistory.js:23]: These types of indications should not be on GitHub.

**Review by YilKanda — COMMENTED** [review]
Remember that your code on GitHub is intended to be in production, therefore it must be kept clean, that is, without code in comments, or draft type indications.

The necessary indications go in the description of the Pull Request, where the work tree is not affected.

**Pull request merged**